codewof/programming/content/en/form-circle/question.md

```diff
@@ -0,0 +1,7 @@
+# Form Circle
+
+A class is outside in a line to play duck duck goose. They must make a circle in order to start.
+
+Write a function `form_circle(students)` that takes a list of students, `students`, and **prints** `{first student} and {last student} should sit next to each to make a circle.`
+
+If there is only one student they should sit next to themselves. Your function will not be tested with 0 students.
```

Comment:
```suggestion
Your function will not be tested with less than 3 students.
```

I think the situations of 1 and 2 students doesn't make much sense, so probably easier to keep the number in tests above 2. The tests would need to be changed too.